src/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+  </packageSources>
+</configuration>
```

Comment:
This file was removed in other changeset likely a merge issue, 
should we removed this file under src and leave only one at the root?

Reply:
Yes, sir good catch.